### PR TITLE
List wheel

### PR DESCRIPTION
**Pull request by Kazumorii**
-Added List Wheel Command
-Removed "Hipefire" from Wheel

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -177,3 +177,249 @@ $RECYCLE.BIN/
...
+# Paket dependency manager
+.paket/paket.exe
+paket-files/
+
+# FAKE - F# Make
+.fake/
+
+# JetBrains Rider
+.idea/
+*.sln.iml
```
Comment (zacleids) [inline .gitignore]: please add a newline at the end of the file. Also, there are many duplicate entries, try to keep the gitignore as small as we can while covering possible files to ignore

**Review by zacleids — CHANGES_REQUESTED** [review]
please remove all of the files in the `.vs` folder

**Comment by Kazumorii** [comment]
Remove .vs file, modified .gitignore to properly ignore the .vs folder.

File: .gitignore [hunk 2]
```diff
@@ -1,179 +1,365 @@
...
-/out/
-
-# mpeltonen/sbt-idea plugin
-.idea_modules/
-
-# JIRA plugin
-atlassian-ide-plugin.xml
-
-# Cursive Clojure plugin
-.idea/replstate.xml
```
Comment (zacleids) [inline .gitignore]: this should all still be in gitignore, because I still use intellij

File: .gitignore [hunk 3]
```diff
@@ -1,179 +1,365 @@
...
-*.msm
-*.msp
+# Ionide (cross platform F# VS Code tools) working folder
+.ionide/
 
-# Windows shortcuts
-*.lnk
+# Fody - auto-generated XML schema
+FodyWeavers.xsd
+/.env
```
Comment (zacleids) [inline .gitignore]: this should just be `.env` not `/.env`

**Review by zacleids — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (zacleids) [reply .gitignore]: all of my original gitignore should be left as is, you should only be adding to it

Comment (Kazumorii) [reply .gitignore]: I think i fucked up my .gitignore

**Comment by Kazumorii** [comment]
Ok, i deleted the .gitignore and reset it to the original project. Thank you for working with me through this as I reorient myself to this.

**Review by zacleids — APPROVED** [review]
LGTM

**Pull request merged**